### PR TITLE
Create Data Model

### PR DESCRIPTION
**Pull request by NurkaAmre**
### In this PR I did following features: 

- Created a new blog-app. 

- Initialized project with Git.

- Setup Postgres database.

- Builded project schema.

- Created and run the necessary migration files.

- Table and column names matched the ERD diagram.

- Foreign keys included.

- All columns that are foreign keys  have a corresponding index.

### CODE REVIEW

File: db/migrate/20230124180818_create_comments.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :author, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```
Comment (ShadyShawkat) [inline db/migrate/20230124180818_create_comments.rb]: - [x] Please make sure that the comment has a foreign key reference to the posts table, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) ERD Diagram.

File: db/migrate/20230124181134_create_posts.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+      t.references :author, null: false, foreign_key: true
+
```
Comment (ShadyShawkat) [inline db/migrate/20230124181134_create_posts.rb]: - [ ] Kindly make sure that the reference foreign key is tied to a table. Here you're referencing a table called "authors" which doesn't exist. (please do this to all tables that have a reference to the "users" table)

- [HINT] You can use the following syntax to do so.
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @NurkaAmre, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AshakaE** [comment]
Hi @NurkaAmre , 

### Highlights ✨ 

- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**